js/style/style.js

```diff
@@ -18,12 +18,13 @@ var QueryFeatures = require('../source/query_features');
 var SourceCache = require('../source/source_cache');
 var styleSpec = require('./style_spec');
 var StyleFunction = require('./style_function');
+var getWorkerPool = require('../global_worker_pool');
```

Comment:
> We could work around this by having the worker pool wait to look up `WORKER_COUNT` until it's actually creating workers during the first `acquire` call.

That sounds like a good compromise to me. Bonus side effect: no workers are created if no maps are created.